tests/test_preprocessing.py

```diff
@@ -114,6 +114,104 @@ def test_pipeline_stopwords(self):
         pipeline = [preprocessing.lowercase, preprocessing.remove_stopwords]
         self.assertEqual(preprocessing.clean(s, pipeline=pipeline), s_true)
 
+    """
+    Test clean
+    """
+
+    def _get_default_clean_pipeline(self):
```

Comment:
Not sure we really need so many tests for this part ...?

Reply:
I think those test will help us to cover all different sections of the pipeline individually, so if something gets changed, we know, which part is broken